### DISCUSSION TITLE
Product module

### DISCUSSION THREAD

**Discussion by EmbedLinuxAcademy** [OE4T/tegra-demo-distro#181] [Q&A]
Hi, I am new here. I am using the tx2-nx module. I tried with demo mender image but the sd card is not working( I want to use it for external storage ). I talked with a product firm but they did not work with yocto. they gave me two dtb files for standard image. Is there any way to use these dtb files with meta-tegra?  How can I solve that issue?

**Comment by ichergui** [comment]
Hello @alpcrypt 

I'm not sure if I understand well your request. `TX2-NX` module doesn't have `SDcard` 
For your test, once you are done with the build you can replace the original dtbs with the new ones then flash your device by using the script `doflash`

**Note:** You can use `dunfell` branch for your test

**Comment by ichergui** [comment]
Hey @alpcrypt 
Any update about this discussion ?
